src/main/kotlin/no/nav/syfo/domain/Vurdering.kt

```diff
@@ -99,6 +101,36 @@ sealed interface Vurdering {
         )
     }
 
+    data class OppfyltUtenForhandsvarsel internal constructor(
+        override val uuid: UUID = UUID.randomUUID(),
+        override val createdAt: OffsetDateTime = nowUTC(),
+        override val personident: PersonIdent,
+        override val veilederident: String,
+        override val begrunnelse: String,
+        override val document: List<DocumentComponent>,
+        override val journalpostId: JournalpostId? = null,
+        override val publishedAt: OffsetDateTime? = null
+    ) : Vurdering {
+        override val type: VurderingType = VurderingType.OPPFYLT_UTEN_FORHANDSVARSEL
+        override val varsel: Varsel? = null
```

Comment:
Ikke egentlig relevant tror jeg.
Er det slik at bruker ikke skal få noe varsel(beskjed/melding) når det sendes en innstilling uten forhåndsvarsel? Vi mangler kanskje litt å kunne representere at det varselet her er egentlig bare journalføringen og ikke et faktisk varsel? 🤔 Jeg blir stadig forvirret av dette 😅

Reply:
Godt poeng, vi må sjekke om det med varsling skal være annerledes. Men kanskje mest aktuelt ved Avslag?

Reply:
Ja 😊 Og er nok ikke noe blocker for dette uansett. Det er jo noe vi kan tilpasse når frontenden etterhvert kommer på plass

Reply:
Betyr det at vi da kan fjerne datafeltet for `OppfyltUtenForhandsvarsel` og en eventuell `AvslagUtenForhåndsvarsel`? 🤔

Reply:
Hm, nei koden kompilerer ikke hvis jeg forsøker å fjerne `varsel` fra feks `OppfyltUtenForhandsvarsel`.